src/libmongoc/CMakeLists.txt

```diff
@@ -45,9 +45,6 @@ if (NOT ENABLE_ZSTD MATCHES "ON|AUTO|OFF")
    message (FATAL_ERROR "ENABLE_ZSTD option must be ON, AUTO, or OFF")
 endif ()
 
-# Disable warnings on bundled zlib source files.
```

Comment:
The debug-compile-with-warnings task is warning on [zlib sources](https://spruce.mongodb.com/task/mongo_c_driver_releng_debug_compile_with_warnings_patch_bebe063a42c87cabd8b628e61e9901da1161a1a7_61a56fb7d6d80a0f59f623ac_21_11_30_00_26_34/logs?execution=0). Should this be added back?

Reply:
Fixed now